dco_check/dco_check.py

```diff
@@ -907,7 +907,7 @@ def get_commit_range(self) -> Optional[Tuple[str, str]]:  # noqa: D102
             return None
         commit_hash_base = None
         commit_hash_head = None
-        if event_name == 'pull_request':
+        if event_name in ['pull_request', 'pull_request_target']:
```

Comment:
I think it would be slightly better to use a tuple instead of a list here.

```suggestion
        if event_name in ('pull_request', 'pull_request_target'):
```

Reply:
Indeed! :100: